extension/podstation.css

```diff
@@ -246,3 +246,68 @@ img.listItemImageSmall {
 #aboutContent {
 	font-size: medium;
 }
+
+.dark-scheme{
+	--dark_text: #d1d2d3;
+	--dark_bg : #333333; /* 1e1e1e*/
```

Comment:
any reason for leaving the comments here?

Reply:
I was just trying two different dark color schemes to see which one looks better. I have removed the comments.
Also at line no 260, I replaced it with variable